src/modules/git_status.rs

```diff
@@ -271,8 +284,8 @@ impl RepoStatus {
         let re = Regex::new(r"branch\.ab \+([0-9]+) \-([0-9]+)").unwrap();
 
         if let Some(caps) = re.captures(s) {
-            self.ahead = caps.get(1).unwrap().as_str().parse::<usize>().unwrap();
-            self.behind = caps.get(2).unwrap().as_str().parse::<usize>().unwrap();
+            self.ahead = Some(caps.get(1).unwrap().as_str().parse::<usize>().unwrap());
+            self.behind = Some(caps.get(2).unwrap().as_str().parse::<usize>().unwrap());
```

Comment:
You can make this safer by using the `parse` return value:
```suggestion
            self.ahead = caps.get(1).unwrap().as_str().parse::<usize>().ok();
            self.behind = caps.get(2).unwrap().as_str().parse::<usize>().ok();
```